src/DocRaptor.php

```diff
@@ -39,15 +40,17 @@ class ApiWrapper
     protected $baseurl;
 
 
-
     /**
+     * @param HttpTransferInterface $httpClient
      * @param string|null $api_key
      */
-    public function __construct($api_key = null)
+    public function __construct(HttpTransferInterface $httpClient, $api_key = null)
```

Comment:
I guess we're starting with a completely fresh user base, also because we're dropping 5.3 support. If we make both params optional, it would be possible to pass null and an API key without changing the order. I don't have a very strong opinion on that so injecting the param first and then the class is fine with me even though I somehow tend to start with higher order types because I think these change less frequently. Anyhoo, I'll make the changes now, let me know what you think.